avm/res/document-db/mongo-cluster/main.bicep

```
@@ -69,6 +69,13 @@ param enableMicrosoftEntraAuth bool = false
 @description('Optional. The Microsoft Entra ID authentication identity assignments to be created for the cluster.')
 param entraAuthIdentities authIdentityType[]?
 
+@description('Optional. Controls public network access to the cluster. Allowed values: "Enabled", "Disabled".')
```

```suggestion
@description('Optional. Controls public network access to the cluster.')
```
The allowed values are anyways communicated via Linter :)